initfiles/bin/init_roxie

```diff
@@ -22,6 +22,9 @@ PID_NAME="$PID/`basename $PWD`.pid"
 INIT_PID_NAME="$PID/init_`basename $PWD`.pid"
 echo $$ > $INIT_PID_NAME
 
+component=`basename $PWD`
+DALISERVER=`$INSTALL_DIR/sbin/configgen -env $CONFIG_DIR/$ENV_XML_FILE -listall | awk 'BEGIN {FS=",";} /DaliServerProcess/ { print $3 }'`
```

Comment:
using -t dali would be a bit more explicit and not require the "/DaliServerProcess/" filter